simplify/src/main/java/org/cf/simplify/Launcher.java

```diff
@@ -179,33 +179,44 @@ private void executeClass(VirtualMachine vm, String className, RuntimeStats stat
                 continue;
             }
 
-            boolean executeAgain;
-            do {
-                System.out.println("(" + stats.getCurrentMethodIndex() + " / " + stats.getTotalMethods() + ") Executing top level method: " + method);
-                ExecutionGraph graph = null;
-                try {
-                    graph = vm.execute(method);
-                } catch (VirtualMachineException e) {
-                    System.err.println("Aborting execution; exception: " + e);
-                }
-
-                if (null == graph) {
-                    System.out.println("Skipping optimization of " + method + "; null execution graph");
-                    stats.incrementFailedMethodCount();
-                    break;
-                }
+            try {
```

Comment:
I will do that tomorrow (in Germany today). Of course it is better to fix the cause of the exceptions and I also realized that it can corrupt the program (the Java VM crashed with a core dump). This is just a quick and dirty fix - which I should mention in the readme.